### ISSUE TITLE
Update persistence.xml

### ISSUE THREAD

**Issue by vonfredd** [Ebyrdeu/movie_jakarta#18]
### Problem Description
There is no possibility to persist data. 
We should update persistence.xml to be able to connect to a database through a datasource.

**Comment by Ebyrdeu** [comment]
It can be related to PR #16 as it kinda of related to docker also

**Comment by vonfredd** [comment]
> It can be related to PR #16 as it kinda of related to docker also

Yes I also think they are related. I just wanted to keep them separated since #16 is focused on docker configs.
